src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -30,6 +28,8 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWithNearbyCommentFilter"/>
```

Comment:
> In the parent-pom

Ok I think I understand.  So, the change of version for com.puppycrawl.tools:checkstyle  in 9789f9f9e0649522d86bb1afc92424d7178e9f16 necessitates this change?  A commit message of `Modified rules to work with checkstyle 8.1` would be a bit more informative.